.github/workflows/build.yml

```diff
@@ -68,6 +68,12 @@ jobs:
         go build -v -o build/tfswitch
         mkdir `pwd`/bin/
         find ./test-data/integration-tests/* -type d -print0 | while read -r -d $'\0' TEST_PATH; do
+            if test -f "${TEST_PATH}/.tfswitch.toml"
+            then
+                cp "${TEST_PATH}/.tfswitch.toml" ~/
+            else
+                { test -f ~/.tfswitch.toml && rm ~/.tfswitch.toml; } || true
+            fi
```

Comment:
What this if-conditional goal is?
If test TOML exists, then copy it to HOME, otherwise delete TOML from HOME 😲

Reply:
Exactly that - it's handled the fact that a test has a TOML file, but we ignore ChDiR (CWD) now, but I wanted to keep the test... so copied to HOMEDIR so it continued to work

Reply:
Ah, I lean to think I got it =)
You should be good to replace `{ test -f ~/.tfswitch.toml && rm ~/.tfswitch.toml; } || true` with a simpler `rm -f ~/.tfswitch.toml`